### PR TITLE
Styles header

### PR DESCRIPTION
**Pull request by AnitaMartinez**
I have added the styles to the header and also I created a nav.

### CODE REVIEW

**Comment by AnitaMartinez** [comment]
Everything looks good

**Review by Ginagonzalezgut — COMMENTED** [review]
It looks great!

File: index.html [hunk 1]
```diff
@@ -7,8 +7,16 @@
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
-    <header>
-      <h1>Trabajo colaborativo con Git</h1>
+    <header class="header">
+      <h1 class="title">Trabajo colaborativo con Git</h1>
+      <nav>
+        <ul>
+          <li><a href="">Item</a></li>
```
Comment (AnitaMartinez) [inline index.html:14]: You've forgotten the links in the `hrf`.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.header {
+  background-color: aquamarine;
```
Comment (AnitaMartinez) [inline css/styles.css:2]: I love this color!

**Review by griseldagomez — APPROVED** [review]
Todo se ve bien

File: css/styles.css (hunk 2 above)
Comment (ShamaGarijo) [reply css/styles.css:2]: **:)******

**Comment by LorenaSTN** [comment]
everything looks good!

File: css/styles.css (hunk 2 above)
Comment (Ginagonzalezgut) [reply css/styles.css:2]: Add a new color

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.header {
+  background-color: aquamarine;
+  padding: 20px;
+}
+
+.title {
+  font-size: 20px;
```
Comment (LorenaSTN) [inline css/styles.css:7]: could you change the font-family?

**Review by ShamaGarijo — CHANGES_REQUESTED** [review]
Add an img

**Review by Ginagonzalezgut — COMMENTED** [review]
Add a new color

**Review by laura-pf — CHANGES_REQUESTED** [review]
change color

File: index.html (hunk 1 above)
Comment (Mariaguerrero95) [reply index.html:14]: Perfect

**Comment by olgabonillo** [comment]
Está todo correcto

File: css/styles.css (hunk 2 above)
Comment (AnitaMartinez) [reply css/styles.css]: Already changed. https://github.com/AnitaMartinez/trabajo-colaborativo-git/pull/1/commits/ea5ee6afbf7b89e03bc2647d002bad86037a4010

**Comment by AnitaMartinez** [comment]
> Add an img

It's not necessary in this step.

File: index.html (hunk 1 above)
Comment (AnitaMartinez) [reply index.html]: https://github.com/AnitaMartinez/trabajo-colaborativo-git/pull/1/commits/1494781e7195ce045149b41428aaacdd34b9255f

**Review by olgabonillo — APPROVED** [review]
Its good!

**Pull request merged**